themes/digital.gov/layouts/partials/core/footer.html

```diff
@@ -77,13 +77,13 @@ <h4 class="usa-footer__primary-link">Social</h4>
                     focusable="false"
                   >
                     <use
-                      xlink:href="{{ "uswds/img/sprite.svg#twitter" | relURL }}"
+                      xlink:href="{{ "uswds/img/sprite.svg#x" | relURL }}"
                     ></use>
                   </svg>
                   <a
                     class="text-ink hover:text-ink text-no-underline hover:text-underline"
                     href="https://twitter.com/digital_gov"
```

Comment:
Same here, `twitter.com`. This one already uses `https`.

Reply:
twitter.com is till the current domain